### ISSUE TITLE
Viewport origin translation is not continuous while applying momentum 

### ISSUE THREAD

**Issue by kirkbyo** [Flutter-Bounty-Hunters/page_list_viewport#37]
**TL;DR**: the refresh rate between frames is not continuous and this is resulting in the origin of the viewport to appear as if it skips during momentum simulation.

While the momentum is being applied, there seems to an occasion frame where it feels like the page skips. When plotting the vertical offset delta that is applied on each frame, you can see there are some sharp changes while the momentum is being applied. 

![239118232-ff328ac6-53b4-4b53-847d-f5592b9f99fa](https://github.com/Flutter-Bounty-Hunters/page_list_viewport/assets/8084674/0359c8f4-53f0-451e-a19f-e98e56519494)

In theory, since the momentum simulation is driven by the `vsync` ticker you'd expect this function to be continuous since it should be ticking at a constant rate.

When capturing the `elapsedTime` and the `origin` for each frame and then capturing the deltas between each frame for both the `elapsedTime` and `origin`, the cause and effect of the discontinuities are clear.

<img width="451" alt="Screen Shot 2023-05-17 at 7 23 02 PM" src="https://github.com/Flutter-Bounty-Hunters/page_list_viewport/assets/8084674/804b794d-02e2-4eeb-ab0b-0f3ebd5a11b8">

_Table which captures a subset of the elapsed time and origin for each frame of a momentum simulation after scrolling in the `demo_release` entrypoint on a 90hz tablet._

Since the tablet the data captures on, refreshes at 90hz, you'd expect a the time delta between each frame to be 1/90 or ~0.011. In the table above you see most frames the elapsed time is ~0.011 (indicated in green), however for some frames, the elapsed time is `0.012` (indicated in red). While in most cases, this would be a rounding error, given the magnitude of our translations and velocity this results in the origin between frames to be discontinuous. For example,

- Frame 21, the origin is -52.5824
- Frame 22, the origin is -55.830
- Frame 23, the origin is -49.810

Given that for the frames leading up to 22, the function is tending towards zero, it is unexpected that the origin in frame 22 is larger than 21 and then begins to increase again in frame 23.

Within the viewport this would appear as if the page jittered.
